decide/base/templates/base.html

```diff
@@ -1,50 +1,52 @@
-        <link rel="preconnect" href="https://fonts.googleapis.com">
-        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Open+Sans&family=Poppins:wght@500&display=swap" rel="stylesheet">        
-        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-        <link rel="stylesheet" href="{% static "base/style.css" %}">
-        {% block extrahead %}{% endblock %}
-    </head>
-    <body>
-        <header class="header">
-            <section class="row">
-                <h1><a id="trapdoor" href="/">Decide</a></h1>
-                <a href="/voting-list">Votaciones</a>
-            </section>
-            <!-- show profile icon if user is logged in -->
-            {% if user.is_authenticated %}
-                <div class="authenticated_icon">
-                    <a href="/logout">Cerrar Sesión</a>
-                    <a class="profile">
-                        <span href="/profile" class="material-icons">account_circle</span>
-                        <span>{{ user.username }}</span>
-                    </a>
-                </div>
-            {% else %}
-                <div>
-                    <a href="/register">Registrarse</a>
-                    <a href="/signin">Iniciar Sesión</a>
-                </div>
-            {% endif %}
-        </header>
-        <section class="content">
-            {% block content %}
-            {% endblock %}
-
-            {% block extrabody %}{% endblock %}
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Decide | {% block title %}{% endblock %}</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Open+Sans&family=Poppins:wght@500&display=swap" rel="stylesheet">        
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="{% static "base/style.css" %}">
+    {% block extrahead %}{% endblock %}
+</head>
+<body>
+    <header class="header">
+        <section class="row">
+            <h1><a id="trapdoor" href="/">Decide</a></h1>
+            <a href="/voting-list">Votaciones</a>
         </section>
-        {% block extrascripts %}
+        <!-- show profile icon if user is logged in -->
+        {% if user.is_authenticated %}
+        <div class="dropdown">
+            <div class="authenticated_icon">
+                <a href="/profile" class="material-icons">account_circle<p>{{user.username}}</p></a></button>
+            </div>
+            <div class="dropdown-content">
+                <a href="/change-password">Cambiar Contraseña</a>
+                <a href="/logout">Cerrar Sesión</a>  
```

Comment:
These links are also affected by the line animation, which we don't want, I suggest you create a class like "header-link" and change the css to point towards that class instead of all "header a". I would also suggest not to use it for the profile because it looks off when the dropdown opens